core/src/com/unciv/ui/objectdescriptions/BaseUnitDescriptions.kt

```diff
@@ -25,15 +25,15 @@ import com.unciv.ui.screens.civilopediascreen.MarkupRenderer
 object BaseUnitDescriptions {
 
     /** Generate short description as comma-separated string for Technology description "Units enabled" and GreatPersonPickerScreen */
-    fun getShortDescription(baseUnit: BaseUnit): String {
+    fun getShortDescription(baseUnit: BaseUnit, exclude: Unique.() -> Boolean = {false}): String {
```

Comment:
instead of 'exclude' please call it 'uniqueExclusionFilter' or somesuch

Same for the rest of these

Reply:
Reminder